scripts/shairport-sync.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=ShairportSync AirTunes receiver
+After=sound.target
+Requires=avahi-daemon.service
+After=avahi-daemon.service
+
+[Service]
+ExecStart=/usr/bin/shairport-sync
```

Comment:
This needs to be adapted to work with the current head of `develop`, see #139 and #134.